toxav/video.c

```diff
@@ -34,6 +34,8 @@
 #include "../toxcore/network.h"
 
 #define MAX_DECODE_TIME_US 0 /* Good quality encode. */
+#define MAX_ENCODE_TIME_US ((1000 / 24) * 1000)
```

Comment:
kk, you're right then, out of scope.

Thanks again for putting so much work into this!